core/modules/SConscript

```diff
@@ -115,6 +115,7 @@ def getSrcs(mName):
     return map(lambda s: os.path.join(mName, s), srcs)
 
 def getTgts(mName):
+    print "Calling getTgts(%s)" % mName
```

Comment:
I would recommend to use the logger here. It can be called this way:
state.log.debug(DEBUG_MSG)